src/views/notification/Edit.vue

```diff
@@ -0,0 +1,286 @@
+                            class="rounded-full"
+                            :src="this.notification.user?.profile_picture ?? require('@/assets/images/avatar.png')"
+                          />
+                        </div>
+                        <div class="ml-5">
+                          <div class="w-24 sm:w-40 truncate sm:whitespace-normal font-medium text-lg">
+                            {{ this.notification.user?.name }}
+                          </div>
+                          <div class="text-gray-600">{{ this?.notification?.user?.role?.name }}</div>
+                        </div>
+                      </div>
+                      <div class="mt-6 lg:mt-0 flex-1 dark:text-gray-300 px-5 border-l border-r border-gray-200 dark:border-dark-5 border-t lg:border-t-0 pt-5 lg:pt-0">
+                        <div class="font-medium text-center lg:text-left lg:mt-3">
+                          Contact Details
+                        </div>
+                        <div class="flex flex-col justify-center items-center lg:items-start mt-4">
+                          <div class="truncate sm:whitespace-normal flex items-center">
+                            <MailIcon class="w-4 h-4 mr-2"/>{{ this?.notification?.user?.email }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <UserIcon class="w-4 h-4 mr-2"/>{{ this?.notification?.user?.pre_name }} {{ this?.notification?.user?.last_name }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <HashIcon class="w-4 h-4 mr-2"/>User ID: {{ this?.notification?.user?.id }}
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="p-5">
+                    <div class="flex flex-col-reverse xl:flex-row flex-col">
+                      <div class="flex-1 mt-6 xl:mt-0">
+                        <div class="grid grid-cols-12 gap-x-5 mb-3">
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Title
+                              </label>
+                              <input
+                                id="update-title-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.title != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Title"
+                                v-model="this.notification.title"
+                              />
+                              <div v-if="this.validation_error?.title != null" class="text-theme-6 mt-2 mb-4">
```

Comment:
`!= null` is unnecessary, questionmark-syntax is taking a valid value as true by default

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/d168bab06f57a70137f9e8dab39812eb5b6ac6a9